README.md

```diff
@@ -11,8 +11,8 @@
 [![GitHub Release Date](https://img.shields.io/github/release-date/stackbuilders/react-native-spotlight-tour)](https://github.com/stackbuilders/react-native-spotlight-tour/releases)
 [![Snyk Vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/@stackbuilders/react-native-spotlight-tour)](https://snyk.io/)
 
-`react-native-spotlight-tour` is a simple and intuitive library for React Native (Android, iOS, and Web 
-compatible). It allows you to implement a highly customizable tour feature with an awesome spotlight 
+`react-native-spotlight-tour` is a simple and intuitive library for React Native (Android, iOS, and Web
```

Comment:
Thanks for the corrections. Maybe we should also explain that the library uses `floting-ui/react-native` and re-exports all their middlewares so they can be used on the configuration, wdyt? 🙂